core/services/helper/main.py

```diff
@@ -93,6 +93,11 @@ class WebsiteStatus(BaseModel):
     error: Optional[str] = None
 
 
+class CockpitWidget(BaseModel):
+    name: str
+    url: str
+
```

Comment:
Thinking on compatibility, it should include a widget version, and a cockpit target version as well

Reply:
yeah, this is the absolutely minimum. I'm thinking of how to expand things without messing up things too much.
would it be an issue if we we had a complex json in here?
should we instead just use this as an aggregator and have it link to another json on the extension?
then we could also manually input that json into cockpit